docs/sources/panels/transformations.md

```diff
@@ -226,6 +226,8 @@ After I apply the transformation, my labels appear in the table as fields.
 
 ### Group By
 
+> **Note:** This documentation refers to a Grafana 7.2 feature.
```

Comment:
This transform is only available in Grafana 7.2+

Reply:
Ah, I just used the same wording as other notes in this md file. But maybe update it on the other places as well?